workers.sample.yml

```diff
@@ -1,44 +1,5 @@
 build_platforms:
-  - 10.5_ppc
-  - 10.5_ppc_legacy
-  - 10.6_x86_64
-  - 10.6_x86_64_legacy
-  - 10.6_i386
-  - 10.6_i386_legacy
-  - 10.7_x86_64
-  - 10.7_x86_64_legacy
-  - 10.8_x86_64
-  - 10.8_x86_64_legacy
-  - 10.9_x86_64
-  - 10.10Î_x86_64
-  - 10.11_x86_64
-  - 10.12_x86_64
-  - 10.13_x86_64
-  - 10.14_x86_64
+  - 10_14_x86_64
```

Comment:
Maybe what we should do is provide two files, `workers.production-sample.yml` and `workers.simple-sample.yml`. I had the same problem that I had to basically delete everything to get it somewhat working.